vunit/sim_if/ghdl.py

```diff
@@ -364,3 +392,60 @@ def simulate(  # pylint: disable=too-many-locals
             subprocess.call(cmd)
 
         return status
+
+    def _compile_source_file(self, source_file, printer):
+        """
+        Runs parent command for compilation, and moves any .gcno files to the compilation output
```

Comment:
Is this specific to GHDL? Do other simulators place the files when desired?

Reply:
I'm not sure, but think that at least Modelsim doesn't have any extra compile-time files generated.